google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -527,13 +544,23 @@ public Boolean getIsLive() {
         return matchesStorageClass;
       }
 
+      public DateTime getCustomTimeBefore() {
+        return customTimeBefore;
+      }
+
+      public Integer getDaysSinceCustomTime() {
+        return daysSinceCustomTime;
+      }
```

Comment:
These names are real confusing to me. Can we rename  or add description on it? for example what is a "customTime"?

Reply:
these name are based on underlying Google API object model so we can't rename it but yes we can add description here what it indicate